dev/com.ibm.ws.security.fat.common.jwt/src/com/ibm/ws/security/fat/common/jwt/utils/JwtTokenBuilderUtils.java

```diff
@@ -167,13 +175,17 @@ public JWTTokenBuilder populateAlternateJWEToken(Key key) throws Exception {
 
     /**
      * Create and add a simple Json payload to the passed in builder
-     * 
+     *
      * @param builder - the builder to upate
      * @return - returns the builder with a simple Json payload (method just creates some random values - currently, no one is looking at them
      * @throws Exception
      */
     public JWTTokenBuilder createAlternateJWEPayload(JWTTokenBuilder builder) throws Exception {
-        // Json Content (buildJWE method will override payload, buildAlternateJWE* methods will use what's set below)
+        return createAlternateJWEPayload(builder, null);
+    }
+
+    public JWTTokenBuilder createAlternateJWEPayload(JWTTokenBuilder builder, List<NameValuePair> extraPayload) throws Exception {
```

Comment:
Would probably make more sense for `extraPayload` to be a `Map<String, String>` object so we're relying on pure Java classes.